packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -214,7 +214,6 @@ export function usePress(props: PressHookProps): PressResult {
           e.preventDefault();
```

Comment:
Do we need to do the check for keyboard events as well?

Reply:
yeah most likely. i'll see if i can add a good test for that